### DISCUSSION TITLE
کار نکردن کانفیگ های ws tls روی نرم افزار sing-box

### DISCUSSION THREAD

**Discussion by bestkilleriranlod** [Gozargah/Marzban#1221] [General]
درود
مشکلی که دارم اینکه کانفیگ هایی که روی کلافلر دارم ws tls روی هسته های sing-box پینگ نمیدن 
علت چی میتونه باشه؟

**Comment by sasanbozorg** [comment]
اگر fingerprint و alpn فعال باشه کار نمیکنه روی sing-box
منم همین مشکلو دارم

**Reply by Mjzaret** [reply]
هسته سینگ باکس با fingerprint مشکلی نداره اما با alpn به مشکل میخوره.
در مورد کانفیگهای بدون tls هم مشکلی که باهاش برخوردم این بود که آپلود صفر هست ولی دانلود و پینگ بسیار خوبی داشتم.
